data/data/azure/dns/variables.tf

```diff
@@ -0,0 +1,40 @@
+variable "tags" {
+  type        = "map"
+  default     = {}
+  description = "tags to be applied to created resources."
+}
+
+variable "cluster_domain" {
+  description = "The domain for the cluster that all DNS records must belong"
+  type        = "string"
+}
+
+variable "cluster_name" {
+  description = "The name for the cluster without the random suffix"
+  type        = "string"
+}
+
+variable "base_domain" {
```

Comment:
https://github.com/openshift/installer/blob/655fdc7818e3fb888d5878f83aaaceb490d86e02/data/data/aws/route53/variables.tf#L17-L20

Reply:
fixed in 9a60194256a9469602406c3548fa6a24519fbb85